### DISCUSSION TITLE
Don't close on mousemove

### DISCUSSION THREAD

**Discussion by Davikar** [tophf/mpiv#93] [MPIV general]
Is it possible to prevent the popup from closing when moving the mouse while previewing videos?
It would make it easier to scroll through videos, as right now I can't reach the seek bar without the popup closing.

**Comment by tophf** [comment]
There are two methods:
1. Hold `Shift`, then seek, release Shift.
2. Zoom the video first.

**Comment by Davikar** [comment]
Is there any way to edit the custom css so I don't have to hold shift (or anything) when playing videos?

**Comment by tophf** [comment]
The [new version](https://github.com/tophf/mpiv/raw/master/script.user.js) has an option to keep playing videos. I'll test it for a while first before publishing.

![image](https://user-images.githubusercontent.com/1310400/155900450-a39754f6-80bb-4659-b23d-40f11de3b320.png)
